apis/exoscale/v1/dbaas_exoscale_kafka.go

```diff
@@ -32,54 +32,54 @@ type ExoscaleKafka struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// Spec defines the desired state of a ExoscaleKafka.
-	Spec ExoscaleKafkaSpec `json:"spec,omitempty"`
+	Spec KafkaSpec `json:"spec,omitempty"`
```

Comment:
Any reason for Renaming the structs here?

Reply:
Not really. I thought it's a lot of duplcation. However, I think it makes more sense to leave it as it was to avoid confusions. Reverted it back to the "old" way.